DependencyInjection/Configuration.php

```diff
@@ -111,7 +111,14 @@ private function getMailersNode()
                     ->end()
                 ->end()
                 ->scalarNode('sender_address')->end()
-                ->scalarNode('delivery_address')->end()
+                ->arrayNode('delivery_addresses')
+                    ->beforeNormalization()
+                        ->always()
+                        ->then(function ($v) { return array_values((array) $v); })
```

Comment:
You don't have to make that change, when the value already is an array:

```php
// ...
->beforeNormalization()
    ->ifString()
    ->then(function ($v) { return (array) $v; })
->end()
```
// ...